src/lighteval/metrics/stderr.py

```diff
@@ -1,3 +1,25 @@
+# MIT License
```

Comment:
Hm not sure we can put it as only for HF team since this file comes almost as is from the Harness, I'd mention their copyright too (The HuggingFace Team + Eleuther AI)